<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoFish</class>
 <widget class="QWidget" name="InfoFish">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Informaci√≥n</string>
  </property>
  <widget class="QLabel" name="nombrePez">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre del pez</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>341</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="imagenInfoFish">
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="infoPez">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>301</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>Informacion del pez</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="margin">
    <number>5</number>
   </property>
  </widget>
  <widget class="QPushButton" name="aceptar">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>460</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
